app/models/user.rb

```diff
@@ -32,4 +32,10 @@ class User < ApplicationRecord
   def confirmation_required?
     created_at < 7.days.ago
   end
+
+  public
+
+  def admin?
```

Comment:
update this method based on user roles

Reply:
for now is it fine?bcz this can be done after user branch merging into developer.